src/Altinn.App.Core/Internal/Process/EventHandlers/ProcessTask/StartTaskEventHandler.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Altinn.App.Core.Features;
+using Altinn.App.Core.Internal.Process.ProcessTasks;
+using Altinn.Platform.Storage.Interface.Models;
+
+namespace Altinn.App.Core.Internal.Process.EventHandlers.ProcessTask
+{
+    /// <summary>
+    /// This event handler is responsible for handling the start event for a process task.
+    /// </summary>
+    public class StartTaskEventHandler : IStartTaskEventHandler
+    {
+        private readonly IProcessTaskDataLocker _processTaskDataLocker;
+        private readonly IProcessTaskInitializer _processTaskInitializer;
+        private readonly IEnumerable<IProcessTaskStart> _processTaskStarts;
+
+        /// <summary>
+        /// This event handler is responsible for handling the start event for a process task.
+        /// </summary>
+        public StartTaskEventHandler(
```

Comment:
This code used to be in DefaultTasksEvents. Common "start task" logic for all process tasks.